### DISCUSSION TITLE
How to use aliases?

### DISCUSSION THREAD

**Discussion by yurkimus** [parcel-bundler/parcel#4704] [General] [answered]
I have `"alias": {
    "styles": "./styles"
  }` in my `package.json`

When I tried to import `.css` files like this `@import 'styles/variables.css';` - it does error `Cannot resolve dependency 'styles/variables.css`.

How to use aliases right?

**Comment by DeMoorJasper** [comment, marked as answer]
Maybe you could try `@import '~styles/variables.css';`

Which Parcel version are you using?

**Reply by yurkimus** [reply]
I use `"parcel-bundler": "^1.12.4"`

'~' helps, thanks. But it looks strange to me. Can I use aliases without it?

p.s.: My IDE can't understand what to do :(
![image](https://user-images.githubusercontent.com/51003204/83951829-119a0780-a83d-11ea-9e7f-cb21603d7bf4.png)

**Reply by DeMoorJasper** [reply]
using node_modules in css is not standard, that's probably why the IDE freaks out.

`~` has nothing to do with aliases, it's just something you need to do in Parcel 1 to use node_modules in css. Aliases are just virtual node_modules. So unfortunately you can't use them without.

I think we dropped the `~` prefix in Parcel 2 but not entirely sure, we just wanted to support the import syntax that gets used by webpack and postcss-import, this is mainly because relative paths in css imports don't have to start with `./` and you could have duplicates...

**Reply by yurkimus** [reply]
Thanks.
